app/javascript/components/widgets/twitter/tweet_profile.jsx

```diff
@@ -7,40 +7,39 @@ const ProfileWrapper = styled.div`
   display: flex;
   flex-wrap: no-wrap;
   justify-content: flex-start;
-  width: 18vw;
+  width: 15vw;
   flex-direction: row;
   align-items: flex-end;
   margin-bottom: ${(props) => (props.primary ? '48px' : `${spacing.xl}`)};
```

Comment:
Is this still needed since the check for `props.primary` has been removed elsewhere?

Reply:
nvmd... I see it is deleted in the next commit. :)